coinstacks/bitcoin/ingester/src/__tests__/tx.test.ts

```diff
@@ -18,7 +18,7 @@ jest.mock('@shapeshiftoss/common-ingester', () => ({
 
 jest.mock('@shapeshiftoss/common-mongo', () => ({
   RegistryService: jest.fn().mockImplementation(() => ({
-    getByAddress: (address: string) => Object.values(db).find((v) => address === v.registration.pubkey), //promise
+    getByAddress: (address: string) => Object.values(db).filter((v) => address === v.registration.pubkey), //promise
```

Comment:
find returns 1 filter can return more than 1. is that okay?

Reply:
yes, that was the intention since we updated `getByAddress` to return an array of documents instead of a single document. fixed broken tests